components/notebook-controller/controllers/notebook_controller.go

```diff
@@ -532,8 +544,14 @@ func generateVirtualService(instance *v1beta1.Notebook) (*unstructured.Unstructu
 	vsvc.SetKind("VirtualService")
 	vsvc.SetName(virtualServiceName(name, namespace))
 	vsvc.SetNamespace(namespace)
-	if err := unstructured.SetNestedStringSlice(vsvc.Object, []string{"*"}, "spec", "hosts"); err != nil {
-		return nil, fmt.Errorf("set .spec.hosts error: %v", err)
+
+	istioHost := os.Getenv("ISTIO_HOST")
+	if len(istioHost) == 0 {
+		istioHost = "*"
+	}
+	if err := unstructured.SetNestedStringSlice(vsvc.Object, []string{istioHost}, "spec", "hosts"); err != nil {
+		return nil, fmt.Errorf("Set .spec.hosts error: %v", err)
+
```

Comment:
I'm doubtful about this. Do we want to pull in anything Istio-related? The changes in manifests do set `ISTIO_HOST`.

From a quick look, it should not be harmful, so we may just as well have this.

Reply:
Hey Jiri! Yes, they have added two new configmap envs the `ISTIO_HOST` and `CLUSTER_DOMAIN` on components/notebook-controller/config/manager/manager.yaml file
I don't think that they are harmful as well. @harshad16 wdyt?